cleanrepo/Program.cs

```diff
@@ -363,6 +387,94 @@ static void RunOptions(Options options)
         Console.WriteLine($"Elapsed time: {stopwatch.Elapsed.ToHumanReadableString()}");
     }
 
+    private static async Task AuditMSDateAccuracy(Options options, DocFxRepo docFxRepo, List<FileInfo> articleFiles)
+    {
+        if (options.DocFxDirectory is null)
+            return;
+        var config = new ConfigurationBuilder()
+            .AddEnvironmentVariables()
+            .Build();
+        string key = config["GitHubKey"]!;
+        IGitHubClient client = IGitHubClient.CreateGitHubClient(key);
+
+        int totalArticles = 0;
+        int freshArticles = 0;
+        int trulyStateArticles = 0;
+        int falseStaleArticles = 0;
+        // This could be configurable in time (or now, even):
+        DateOnly staleContentDate = DateOnly.FromDateTime(DateTime.Now.AddYears(-1));
+
+        var linkedArticles = from article in articleFiles
```

Comment:
Why are you only checking files that are linked from the TOC? I mean, yes, they all should be, but that seems outside the scope of this function.

Reply:
Well, I was trying to make this function run faster. I removed it, and the elapsed time is more or less the same. I removed it.